_tuning-your-cluster/availability-and-recovery/workload-management/workload-management.md

```diff
@@ -0,0 +1,130 @@
+ 
+- Tenant-level admission control and reactive query management. It can identify and cancel resource-intensive queries when configured thresholds are exceeded, ensuring fair resource allocation.
+
+- Tenant-level isolation within the cluster for search workloads, operating at a node level.
+
+System administrators can dynamically manage query groups using the Workload management APIs. These query groups can be used to make search requests with resource limits.
+
+### QueryGroup
+
+```json
+{
+  "_id" : "16YGxFlPRdqIO7K4EACJlw",
+  "name" : "ping",
+  "resiliency_mode" : "soft",
+  "resource_limits" : {
+    "cpu" : 0.3,
+    "memory": 0.2
+  },
+  "updated_at" : 1729814077916
+}
+```
+
+## Feature operating modes
+
+Query group mode determines the operating level of the feature and it has the following operating modes.
+- **Disabled mode** -- It means the feature will not work at all.
+- **Enabled mode** -- It means the feature is enabled and will cause cancellations and rejection once the query group's configured thresholds are breached.
+- **Monitor_only mode**(Default) -- It means the feature will run and monitor the tasks but it will not cancel/reject the queries .
+
+These modes can be controlled and changed using `_cluster/settings` endpoint with `wlm.query_group.mode` setting.
+
+### Workload management settings
+There are following settings can be used to customize Workload management.
+
+| **Setting name**  | **Description**  |
+| :--- | :--- |
+| `wlm.query_group.duress_streak` | Determines the node duress threshold. Once the threshold is reached, the node is marked as `in duress`. |
+| `wlm.query_group.enforcement_interval`  | Defines the monitoring interval. |
+| `wlm.query_group.mode`  | Defines the feature operating mode. |
+| `wlm.query_group.node.memory_rejection_threshold` | Defines the query group level `memory` threshold. When the threshold is exceeded, the request is rejected.  |
+| `wlm.query_group.node.cpu_rejection_threshold` | Defines query group level `cpu` threshold.  When the threshold is exceeded, the request is rejects. |
+| `wlm.query_group.node.memory_cancellation_threshold` | Controls whether the node is considered in duress when the `cpu` threshold is reached and the effective request cancellation threshold based on `memory` usage. |
+| `wlm.query_group.node.cpu_cancellation_threshold`    | Controls whether the node is considered in duress when the `cpu` threshold is reached and the effective request cancellation threshold on `cpu` usage. |
+ 
+When setting rejection andcancellation settings thresholds, remember that the rejection threshold for a resource should always be less than the cancellation threshold. 
+
+### Workload management stats
+
+The stats API is useful to gather current workload management metrices at query group level. The stats API looks like following
```

Comment:
Workload stats API?

Reply:
these stats are specific to this feature, workload stats is a very broad term and implicitly refers to indexing and search workload. Here we just mean the tenant level search workload stats.